build/admin-page.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array('wp-polyfill'), 'version' => '6d8d3cc10bb4327a5d847f75b4f497e1');
```

Comment:
This is a file that's generated by `wp-scripts build` process...

Looks like it's a way to dynamically include the dependencies on the WP side:
https://developer.wordpress.org/block-editor/reference-guides/packages/packages-dependency-extraction-webpack-plugin/#wordpress

I'll give it a try to see if it's worth the "magic" to avoid having to maintain the dependency list or not.

Reply:
ok, magic invoked in a2131bec664d422e3e9d578da4ab7c6c297b2001.

Pretty cool.  It lets us use `import` syntax and not have to manage dependencies inside the PHP side of the plugin.